### ISSUE TITLE
wiz_project: undefining risk_profile in a resource results in perpetual change detection

### ISSUE THREAD

**Issue by gramsa49** [AxtonGrams/terraform-provider-wiz#54]
### Terraform Version and Provider Version
```
Terraform v1.3.5
on linux_amd64
+ provider registry.terraform.io/axtongrams/wiz v1.0.8
```

### Affected Resource(s)
- wiz_project

### Terraform Configuration Files
Create a resource using this HCL:
```hcl
resource "wiz_project" "test1" {
  name     = "Terraform Test"
  description = "Testing new wiz_project Terraform resource parameters"
  risk_profile {
    business_impact = "MBI"
  }
}
```

Update the resource as follows (remove risk_profile):
```hcl
resource "wiz_project" "test1" {
  name     = "Terraform Test"
  description = "Testing new wiz_project Terraform resource parameters"
}
```

Re-apply the changes and the following is returned:
```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # wiz_project.test1 will be updated in-place
  ~ resource "wiz_project" "test1" {
        id          = "91dd82a0-5521-40f3-8ac5-f42a191e2555"
        name        = "Terraform Test"
        # (3 unchanged attributes hidden)

      - risk_profile {
          - business_impact       = "MBI" -> null
          - has_authentication    = "UNKNOWN" -> null
          - has_exposed_api       = "UNKNOWN" -> null
          - is_actively_developed = "UNKNOWN" -> null
          - is_customer_facing    = "UNKNOWN" -> null
          - is_internet_facing    = "UNKNOWN" -> null
          - is_regulated          = "UNKNOWN" -> null
          - regulatory_standards  = [] -> null
          - sensitive_data_types  = [] -> null
          - stores_data           = "UNKNOWN" -> null
        }

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

The update to remove the risk_profile did not effectively remove the risk_profile parameters.

### Expected Behavior
The risk profile parameters should have been undefined when removed from the resource block.

### Actual Behavior
The risk profile parameters were not undefined when removed from the resource block.

### Important Factoids
This is most likely an issue in the json marshalling of the request body, where empty values are omitted and the API handles this as a no-op instead of undefining the .

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by david-at-appomni** [comment]
Hey there - I'm running into this @ 1.2.5, even though this was closed in #47 

```
  # wiz_project.appomni-corp-gcp-org will be updated in-place
  ~ resource "wiz_project" "my-jell-org" {
        id                 = "org-id"
        name               = "Jelly Org Tenant"
        # (7 unchanged attributes hidden)

      - risk_profile {
          - business_impact       = "MBI" -> null
          - has_authentication    = "UNKNOWN" -> null
          - has_exposed_api       = "UNKNOWN" -> null
          - is_actively_developed = "UNKNOWN" -> null
          - is_customer_facing    = "UNKNOWN" -> null
          - is_internet_facing    = "UNKNOWN" -> null
          - is_regulated          = "UNKNOWN" -> null
          - regulatory_standards  = [] -> null
          - sensitive_data_types  = [] -> null
          - stores_data           = "UNKNOWN" -> null
        }
    }
```

Should I make a new issue?

Hacky workaround:
```
  lifecycle {
    # wiz_project resources struggles with the default for the risk_profile block
    # so we need to ignore changes to the risk_profile block if we are using defaults
    # https://github.com/AxtonGrams/terraform-provider-wiz/issues/54#issuecomment-3300149939
    ignore_changes = [risk_profile]
  }
```

**Comment by jschoombee** [comment]
Yes please do create a new issue for this and share steps and expected behaviour. It can reference this issue.

Out of interest - any reason for not using the tf provider for project resources?
